pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -39,6 +39,9 @@ const (
 	FindVMByUUID FindVM = iota // 0
 	FindVMByName               // 1
 
+	POOL_SIZE  int = 8
+	QUEUE_SIZE int = POOL_SIZE * 10
```

Comment:
These two consts are also defined as `cm.POOL_SIZE` and `cm.QUEUE_SIZE`. I think you can use them directly rather than redefining them here since you are importing `pkg/common/connectionmanager`.

Reply:
Not needed. Has been removed.